Gemfile

```diff
@@ -36,6 +36,7 @@ gem 'lograge'
 gem 'logstash-logger'
 gem 'jbuilder'
 gem 'paperclip'
+gem 'maxminddb'
```

Comment:
IIRC maxminddb needs frequent updating to remain current -- are we going to reflect that with a script on the ops side?